hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/SnapshotChainManager.java

```diff
@@ -71,7 +71,27 @@ public SnapshotChainManager(OMMetadataManager metadataManager) {
   private void addSnapshotGlobal(UUID snapshotID,
                                  UUID prevGlobalID) throws IOException {
     // On add snapshot, set previous snapshot entry nextSnapshotID = snapshotID
```

Comment:
This comment doesn't belong here. Please either remove this or move it before line number 85 (may be).

Reply:
done